### DISCUSSION TITLE
unable to assign *.now.sh as the domain for my vercel project

### DISCUSSION THREAD

**Discussion by 5R33CH4** [vercel/community#52] [Help] [answered]
unable to assign *.now.sh as the domain for my vercel project
It says it is invalid
pls help

**Comment by leerob** [comment, marked as answer]
All .now.sh domains are redirected to .vercel.app - you should use that instead 👍🏼 If there’s an error, it’s likely because the slug has already been taken, so then you would need to try a different one (e.g. my-new-site.vercel.app).
